### ISSUE TITLE
removing gradients with the optimizer

### ISSUE THREAD

**Issue by mynameisvinn** [marrrcin/pytorch-resnet-mnist#2]
during the training loop (cell 23), you do `model.zero_grad()`. i think it should be `optimizer.zero_grad()`.
